### ISSUE TITLE
Dear reviewer

### ISSUE THREAD

**Issue by rmarx** [rmarx/holblocking-blogpost#1]
Thank you so much for volunteering to beta-read yet another one of my way too long blogposts. Remind me to buy you a beverage of your choice the next time we meet! 

This time, I'm taking the approach of having the README.md in this repository just contain the entire text. I hope this allows full flexibility for you to leave your comments. Feel free to just post them in this issue thread, create new issues for larger parts or create a PR for the README file and leave comments in-line like that (however, please do not create a new issue for each individual typo 😛).

Feedback that I'm mainly looking for in order of importance:

- logical errors and false statements
- missing parts that really should be included
- unclear parts that require additional examples/clarification
- parts that can be trimmed / left out (this post is quite bulky) (e.g., I'm not sure about the whole discussion on HTTP/2 performance and congestion control? Maybe that could be a Bonus section as well?)
- Speaking of those: what do you think of the approach of offloading interesting but not core content to bonus section at the end?
- missing references (and references that are better than the ones I have now)
- What technical level would you say this post is intended for (junior/medior/senior/expert)? Put differently: what technical level are you yourself at and what (if anything) did you learn from this post?
- spelling and grammatical errors (though I will re-read myself at least 3 times as well)

Let out your inner pedant! Bad feedback is also feedback! I have a tough skin! I'd rather go cry in a corner than make false claims!

I'm looking forward to hearing what you think!

**Comment by andydavies** [comment]
> HTTP/3 is promising major performance improvements compared to HTTP/2, mainly because it changes its underlying transport protocol from TCP to QUIC

Should this say UDP rather than QUIC?

**Comment by andydavies** [comment]
Typo in HOL blocking in HTTP/3 over QUIC secrion

qutie -> quite

**Comment by djc** [comment]
`even a single issue at he "head of the line" can "block"` <- "he" -> "the"

**Comment by djc** [comment]
I would abbreviate Head of Line as HoL rather than HOL, might be a stylistic nit (not a native speaker, so maybe ask someone who is).

**Comment by djc** [comment]
"is another layer in between HTTP and TCP for security with the TLS protocol." seems like a bit of an awkward sentence, maybe reword/re-order?

I would probably spend much less text (and potentially no pictures at all) on how HTTP/1.1 doesn't support multiplexing. This seems to follow very straightforwardly from how the protocol works, if you have any understanding of HTTP (which you probably need to have anyway in order to make sense of this post?). (Potentially interesting meta-question which could also be helpful to have in the article's introduction: who is this article for?)

**Comment by djc** [comment]
I also wonder about the choice to explain HoL with file/response size issues. I think the packet loss angle makes things quite a bit clearer. See the animation I made:

![hol](https://user-images.githubusercontent.com/158471/83407042-355de780-a410-11ea-8df3-66363707d290.gif)

**Comment by djc** [comment]
Isn't 0-RTT also available on HTTP/2 over TLS 1.3?

`Unlike HTTP/1.1, HTTP/2 puts a so-called DATA frame in front of each chunk.` reads weird to me because to me conceptually the chunk is part of the DATA frame -- isn't that kind of what a frame is?

**Comment by LPardue** [comment]
> The blogpost doesn't really have a publishing deadline, but it would be great to have all comments by next week Friday (May 5) so it can be posted sometime the week of May 8.

Meta review comment, your dates are wrong :P

**Comment by djc** [comment]
While the pipelining appendix seems useful/on-topic, I feel like the Congestion Control one is much less so. In particular, the last section on relative performance of HTTP/2 vs HTTP/1.1 is kind off on a tangent.

**Comment by djc** [comment]
(Also meta-review comment: perhaps structure this as a PR where people can put feedback? That would make it easier to point directly into the text, add suggestions for typos etc, and comment on issues throughout the text in one go while keeping the feedback attached to particular places.)

**Comment by andydavies** [comment]
"A good real-life metaphore" -> "A good real-life metaphor"
